playground/src/pages/CodingEnvironment.tsx

```diff
@@ -260,9 +262,13 @@ export default function CodingEnvironment(): JSX.Element {
       {showDebug ? (
         <div className="flex justify-center pt-6">
           <DebugButton
-            icon="PPrevious"
+            icon="Start"
             onClick={() => executeDebug(DebugCommand.RewindAll)}
           />
+          <DebugButton
+            icon="PPrevious"
+            onClick={() => executeDebug(DebugCommand.Rewind, BigInt(100))}
```

Comment:
It is unclear what this does from the icons alone, please add a title to eac button, suggestions:

- `Go to start`
- `100 steps backwards`
- `step backwards`
- `step`
- `100 steps`
- `Go to end`